guides/common/modules/proc_managing-the-lifecycle-of-the-content-view.adoc

```diff
@@ -3,10 +3,10 @@
 
 The default location of any new Content View is in the Library Environment.
 Optionally, you can add a new environment and promote your Content View to it.
-Use the following procedure to create a new Lifecycle Enviroment.
+Use the following procedure to create a new Lifecycle Environment.
```

Comment:
Create implies new. This is anti-minimalist. 
Look at how you capitalised lifecycle environment one way and now another way in this PR. Please be consistent.

Reply:
Changed to be consistent with rest of documentation.